entry_types/paged/packages/pageflow-paged/src/frontend/phoneLandscapeFullscreen.js

```diff
@@ -1,16 +1,14 @@
-pageflow.phoneLandscapeFullscreen = function() {
-  if (window.screen.orientation) {
-    pageflow.ready.then(function() {
```

Comment:
Why was ready removed here?

Reply:
this function is already called from within ready.then

Reply:
Ok. But as a general advice, when performing such large scale renaming/restructuring, I'd try to separate the mechanical/query replace based changes as much as possible from logic changes since those are otherwise really hard to spot among the many boring changes.